scripts/mirror.js

```diff
@@ -138,6 +145,46 @@ const getSource = (browser, forced_source) => {
   return source;
 };
 
+/**
+ * @param {SupportStatement} compatData
+ * @param {string | null} versionToCheck
+ * @return {string | null}
+ */
+const isVersionAdded = (compatData, versionToCheck) => {
+  if (typeof compatData.version_added === 'string')
+    return compatData.version_added == versionToCheck;
+
+  if (compatData.constructor === Array) {
+    for (var i = compatData.length - 1; i >= 0; i--) {
+      if (compatData[i].version_added == versionToCheck) {
+        return true;
```

Comment:
How about https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some instead of this loop?

Reply:
I keep forgetting `Array.some()` is a thing... :P